bluepyopt/ephys/create_acc.py

```diff
@@ -0,0 +1,349 @@
+'''create a hoc file from a set of BluePyOpt.ephys parameters'''
```

Comment:
I guess this documentation string needs to updated?

Reply:
Yes, thanks, should be resolved with the latest commit